Nodejs/Product/Nodejs/NodejsPackage.cs

```diff
@@ -241,6 +241,8 @@ internal sealed partial class NodejsPackage : CommonPackage {
             MakeDebuggerContextAvailable();
 
             IntellisenseOptionsPage.AnalysisLogMaximumChanged += IntellisenseOptionsPage_AnalysisLogMaximumChanged;
+            IntellisenseOptionsPage.AnalysisLevelChanged += IntellisenseOptionsPageAnalysisLevelChanged;
+            IntellisenseOptionsPage.SaveToDiskChanged += IntellisenseOptionsPageSaveToDiskChanged;
```

Comment:
Can you verify that LoadSettingsFromStorage executes before these event subscriptions? Otherwise the TelemetryLogger will log the IntelliSenseLevelChanged event every time the package is loaded.